src/views/components/pyi-escape-m2b-form.njk

```diff
@@ -0,0 +1,65 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+<form id="pyiEscapeM2bForm" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+        {% if context === "abandon" %}
+          {% set radiosConfig = {
+          idPrefix: "journey",
+          name: "journey",
+          fieldset: {
+            legend: {
+              text: 'pages.pyiEscapeM2b.content.subHeading3' | translate,
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          items: [
+            {
+              value: "end",
+              text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton1Text' | translate
+            },
+            {
+              value: "next",
+              text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton2Text' | translate
+            }
+          ]
+        } %}
+        {% else %}
+          {% set radiosConfig = {
+            idPrefix: "journey",
+            name: "journey",
+            fieldset: {
+              legend: {
+                text: 'pages.pyiEscapeM2b.content.subHeading3' | translate,
+                classes: "govuk-fieldset__legend--m"
+              }
+            },
+            items: [
+              {
+                value: "end",
+                text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton1Text' | translate
+              },
+              {
+                value: "next",
+                text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton2Text' | translate
+              },
+              {
+                divider: 'pages.pyiEscapeM2b.content.formRadioButtons.separateOptionsInFormText' | translate
+              },
+              {
+                value: "next/bank-account",
+                text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton3Text' | translate
+              }
```

Comment:
Is there not a way we can add extra items to the radios, rather than having to duplicate?
I vaguely remembered `push` is a thing in nunjucks - https://giuliachiola.dev/posts/add-items-to-an-array-in-nunjucks/